NetUtils.py

```diff
@@ -79,6 +79,7 @@ class NetworkItem(typing.NamedTuple):
     item: int
     location: int
     player: int
+    """ sending player, except in LocationInfo (from LocationScouts) when it is the receiving player """
```

Comment:
I personally don't like the spaces, but can accept that. Though since this is a full sentence now, it should use proper grammar.
```suggestion
    """ Sending player, except in LocationInfo (from LocationScouts), when it is the receiving player. """
```

Reply:
For useless statistics, outside of `worlds/`, 93.21% have no spaces, 6.27% use them. If you're wondering where that other half a percent went, that's because there are two cases that only have spaces on one side